store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/FederatedOperationChainValidator.java

```diff
@@ -107,10 +106,7 @@ protected void validateViews(final Operation op, final User user, final Store st
                 if (graphIdValid) {
                     currentResult = new ValidationResult();
                     clonedOp.graphIdsCSV(graphId);
-                    // Deprecated function still in use due to Federated GetTraits bug with DYNAMIC_SCHEMA
-                    if (!graphSerialisable.getGraph().getStoreTraits().contains(StoreTrait.DYNAMIC_SCHEMA)) {
-                        super.validateViews(clonedOp, user, store, currentResult);
-                    }
+                    super.validateViews(clonedOp, user, store, currentResult);
```

Comment:
So before this would only validate views for proxy and federated stores, but now will for all stores?
Is this okay?

Reply:
Yes, I think so. like what's happening in your other comment, You get things like validation happening further up the chain. So Proxy and FederatedStore now will Masquerade more close as the graphs you've asked for. 

this dynamic schema was a cheap defeat for when Schema and Traits didn't behave with Federation and Proxy.